API.pm

```diff
@@ -218,6 +218,22 @@ sub getArtist {
 	});
 }
 
+sub getLabel {
+	my ($self, $cb, $labelId) = @_;
+
+	$self->_get('label/get', sub {
+		my $results = shift;
+
+		$results->{albums}->{items} = _precacheAlbum($results->{albums}->{items}) if $results->{albums};
+
+		$cb->($results) if $cb;
+	}, {
+		label_id => $labelId,
+		extra     => 'albums',
+		limit     => QOBUZ_DEFAULT_LIMIT,
```

Comment:
Interesting how I look more critically at others' code than I do at my own... I wanted to ask for validation of `$labelId`, just to figure out that I'd never validate any input in these calls either...

Reply:
So leave as it is? :)

Reply:
Yeah, just leave it as it is.